privaterelay/settings.py

```diff
@@ -153,6 +153,7 @@
 AWS_SQS_QUEUE_URL = config("AWS_SQS_QUEUE_URL", None)
 
 RELAY_FROM_ADDRESS = config("RELAY_FROM_ADDRESS", None)
+NEW_RELAY_FROM_ADDRESS = config("NEW_RELAY_FROM_ADDRESS", None)
```

Comment:
Nit: I know we use this pattern a lot, but it seems the code is broken if `NEW_RELAY_FROM_ADDRESS` isn't in the environment. If we used this, then that would be a startup error `decouple.UndefinedValueError: NEW_RELAY_FROM_ADDRESS not found. Declare it as envvar or define a default value.` rather than a "why are things not working?" error:

 ```suggestion
NEW_RELAY_FROM_ADDRESS = config("NEW_RELAY_FROM_ADDRESS")
```

Reply:
@maxxcrawford I suspect you hit an issue with this on Thursday. Was there a useful error?

Reply:
I did! I've ran across it a few times, so I knew what to expect.

I mentioned it in Slack because I had multiple other engineers reach out to me directly about debugging it.